FirebaseCore/Sources/Public/FirebaseCore/FIROptions.h

```diff
@@ -60,6 +60,7 @@ NS_SWIFT_NAME(FirebaseOptions)
  */
 @property(nonatomic, copy) NSString *GCMSenderID NS_SWIFT_NAME(gcmSenderID);
 
+// TODO(v9): Can this be non-null.
```

Comment:
Is there a historical reason as to why `projectID` is `nullable`? While re-working this PR, I saw some usage of it (in AppCheck sdk for example) that pretty much relies on the `projectID` being there for the SDK to function. I was wondering if this property could be made non-null in v9?



@paulb777 
@chliangGoogle 
@ryanwilson